.github/verify_header.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+#
+# Copyright(c) 2022 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+YEAR=$(date +"%Y")
+REGEX="Copyright\(c\) [0-9]{4}-([0-9]{4}) |Copyright\(c\) ([0-9]{4}) "
+
+while read -r line; do
+    if [[ "$line" =~ $REGEX ]]; then
+        echo ${BASH_REMATCH[0]}
+        if [[ $YEAR == ${BASH_REMATCH[1]} || $YEAR == ${BASH_REMATCH[2]} ]]; then
+            echo $1 have appropriate license header
+            exit 0
+        fi
+        continue
+    fi
+done < "$1"
+
+ echo $1 have wrong license header year
```

Comment:
I'm just wondering why you moved this message outside the loop. In current implementation we don't actually know what went wrong here - if just the year was not set properly, or if the license header haven't been found at all. In the code that I suggested, this message was in place where `continue` is now, and that way we knew that header was found but with a wrong year.